src/utils/environment_helpers.py

```
@@ -0,0 +1,70 @@
+"""This module contains functions to read in environment variables"""
+
+import base64
+import logging
+import os
+
+import boto3
+
+LOGGER = logging.getLogger()
+LOGGER.setLevel(logging.INFO)
+
+
+def validate_env_variable(env_var_name):
+    print(f"Getting the value of the environment variable: {env_var_name}")
+
+    try:
+        env_variable = os.environ[env_var_name]
+    except KeyError:
+        raise Exception(f"Please, set environment variable {env_var_name}")
+
+    if not env_variable:
+        raise Exception(f"Please, provide environment variable {env_var_name}")
+
+    return env_variable
+
+
+def get_database_password():
+    try:
+        password = validate_env_variable("DATABASE_PASSWORD")
+        return password
+    except Exception:
+        pass
```

Don't like this -- we're silently eating all exceptions from the db password being absent?

Can we either:
1) create a named exception and test for that, or
2) explicitly check if DATABASE_PASSWORD is empty/absent and continue if it is?

---

Have just removed DATABASE_PASSWORD and its associated logic and test. Only added for tests but we can and do already get around it with moto's mock_secretsmanager